### PR TITLE
fold in community article and remove link.

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-75d1d6466c/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-7d65f9cf3b/how-to/)

**Comment by cyrichardson** [comment]
@catlook Suggestions here: 
[remove-vpn-user-myrackspace-portal_cyrEdits2.docx](https://github.com/rackerlabs/rackspace-how-to/files/2877487/remove-vpn-user-myrackspace-portal_cyrEdits2.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1880909a93/how-to/). :bow:

File: content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md [hunk 1]
```diff
@@ -1,49 +1,58 @@
...
+Linux&reg;, and security devices as well as network devices.
 
 ### Remove VPN user
 
 To remove the VPN user, use the following steps:
 
 1. Log in to the [MyRackspace customer portal](https://my.rackspace.com/portal/auth/login).
 
-   You will need your Rackspace account number, as well as your username and password.
+   You need your Rackspace account number, as well as your username and password.
```
Comment (WilliamLoy) [inline content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md]: You only need a username and password. This was changed when all of the login portals were consolidated. Additionally I would recommend removing this line all together as the user will see that they need a username and password once they get to the long page.

File: content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md [hunk 2]
```diff
@@ -1,49 +1,58 @@
...
 
-**Warning**: A VPN user who is removed by following these instructions is also deleted from all Windows&reg;, Linux&reg;, and security devices as well as network devices.
+**Warning**: A VPN user who is removed by following these instructions is also deleted from all Windows&reg;, 
+Linux&reg;, and security devices as well as network devices.
 
 ### Remove VPN user
 
 To remove the VPN user, use the following steps:
 
 1. Log in to the [MyRackspace customer portal](https://my.rackspace.com/portal/auth/login).
```
Comment (WilliamLoy) [inline content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md]: Recommend using login.rackspace.com

File: content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md (hunk 1 above)
Comment (catlook) [reply content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md]: Doh!!  Good catch!

File: content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md (hunk 2 above)
Comment (catlook) [reply content/dedicated-hosting/remove-vpn-user-myrackspace-portal.md]: roger!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e534e25900/how-to/)

**Pull request merged**